playground-common/src/Playground/Types.hs

```diff
@@ -110,9 +110,13 @@ type SimulatorAction = ContractCall (Fix FormArgumentF)
 
 type Expression = ContractCall JSON.Value
 
+-- | The example simulations have meaningful (hard-coded) names, but, for any simulation the
+-- user creates in the playground, simulationName = "Simulation " <> show simulationId;
+-- the simulationId is simply present to ensure every simulation has a unique name.
```

Comment:
Real nitpick, but to futureproof against our proposed changes can we say, "the simulationId is simply present to ensure every simulation **is created with** a unique name."